samples/Makefile.am

```diff
@@ -9,57 +9,24 @@ samplessysconfdir = $(sysconfdir)
 
 samplesconf_SCRIPTS = \
 	swtpm-create-tpmca \
-	swtpm-create-user-config-files
+	swtpm-create-user-config-files \
+	swtpm-localca
```

Comment:
Reverted to the state prior to the `samples/swtpm-localca.2inst` hacks introduced [here](https://github.com/stefanberger/swtpm/commit/50f7d49ad3aa92117a9f4701f315c6d567aa21bb).